### ISSUE TITLE
Accessibility: Missing VoiceOver support

### ISSUE THREAD

**Issue by kuestenjunge** [signalapp/Signal-iOS#5045]
- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
If I'm using VoiceOver I can't use Signal.
No element is accessible by VoiceOver, nothing is read out by VoiceOver.

### Steps to reproduce
- close Signal App
- Turn on VoiceOver support on iOS
- open Signal App

**Actual result:**
Nothing is read by VoiceOver
No button, chat, message is accessible. 

**Expected result:** 
 Fully accessible app by VoiceOver 

### Device info

**Device**: iPhone 12 Pro 

**iOS version**: 14.6

**Signal version:** 5.15.1.3

**Comment by kuestenjunge** [comment]
I disabled all features on VoiceOver Recognition in VoiceOver settings.
Signal works like before and I can use it with VO.

I enabled all settings in VOR like before and Signal works fine with VoiceOver on.

**Comment by kuestenjunge** [comment]
It seems that voiceover recognition setting of screen recognition was enabled for signal.￼
